ui/src/common/QualityInfo.js

```diff
@@ -1,14 +1,24 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import Chip from '@material-ui/core/Chip'
 import { LOSSLESS_FORMATS } from '../consts'
 
-export const QualityInfo = ({ record, ...rest }) => {
+export const QualityInfo = ({ record, size, ...rest }) => {
   let { suffix, bitRate } = record
   suffix = suffix.toUpperCase()
   let info = suffix
   if (!LOSSLESS_FORMATS.includes(suffix)) {
     info += ' ' + bitRate
   }
+  return <Chip size={size} variant="outlined" label={info} />
```

Comment:
You are still missing the {...rest} here:
```js
return <Chip variant="outlined" {...rest} label={info} size={size} />
```

Reply:
Ok will implement that but we are nowhere customizing the chip component. (Or is it if we want to in future?)

Reply:
I agree this is a premature introduction of customization, and can be seen as "dead code". But it is such a small change, and something that is extensively used in React-Admin and Navidrome, that I rather have it in place. Specially for components that are in the `common` folder (meaning they can be reused in other scenarios not originally envisioned. We may show this `Qualityinfo` component in other places and adding this is helpful.